test/tests/concrete_ASR_swelling/tests

```diff
@@ -51,4 +51,41 @@
     custom_cmp = 'asr_confined.cmp'
     prereq = test_full
   [../]
+
+  [./ASRtest1_case1]
+    type = 'Exodiff'
+    input = 'asr_validation.i'
+    exodiff = 'asr_validation_case1_out.e'
+    cli_args = 'Outputs/file_base=asr_validation_case1_out Postprocessors/error_ASR/type=ElementL2Error Postprocessors/error_ASR/variable=ASR_vstrain Postprocessors/error_ASR/function=strain_function Postprocessors/error/type=ElementL2Error Postprocessors/error/variable=volumetric_strain Postprocessors/error/function=strain_function'
```

Comment:
It looks like we don't have the checks turned on in BlackBear yet, but we need to add 'requirement', 'issues', and 'design' parameters for these tests.

Also, how long do they take to run? These are validation tests even though they run fast. I'm kind of on the fence about whether to include them in the regression tests or in assessment. We should at least put them in their own directory and include an md file that describes the tests and points to the paper that we are comparing against.

Reply:
These tests are fairly fast, it takes less than 1.5 sec each. I will add documentation and SQA tags. Yes, let's talk about how to maintain these validation cases.

Reply:
OK, in that case, I think they should be in the 'tests' directory, but in a different directory from this one.  Maybe 'concrete_ASR_swelling_validation'?

Reply:
Okay, sounds good.